components/x-eventpromo/src/Image.scss

```diff
@@ -0,0 +1,58 @@
+@import 'styles/bootstrap';
+
+.img {
+  position: relative;
+  top: 0;
+  right: 0;
+  width: 100%;
+
+  @include oGridRespondTo($from: S) {
+	width: 340px;
+	z-index: -1;
+  }
+
+  @include oGridRespondTo($from: M) {
+	width: 380px;
+  }
+
+  @include oGridRespondTo(L, XL) {
+	width: 300px;
+  }
+}
+
+/* z-index setting because o-header is 103 !important */
+.fade-0 {
+	opacity: 1;
+	z-index: 10;
+}
+.fade-1 {
+	opacity: 0;
+	z-index: 20;
+	animation: fade 20s 2s infinite;
+}
+.fade-2 {
+	opacity: 0;
+	z-index: 30;
+	animation: fade 20s 6s infinite;
+}
+.paused {
+	animation-play-state: paused;
+}
+@keyframes fade {
```

Comment:
Worth checking if this animation name gets processed or not. If not this needs a better namespaced name.

Reply:
can't find any documentation on this but inspecting the output, looks like the animation names are scoped